src/Microsoft.DotNet.ProjectModel/FileSystemGlobbing/HashCodeCombiner.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
HashCodeCombiner is also used by DependencyModel.  I think this code will need to go into that assembly instead.